### DISCUSSION TITLE
Verify Faces from a Given Image 这个功能如何请求

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#536] [Q&A]
我按照文档通过PostMan请求了
/api/v1/recognition/faces/91307f12-a92f-430b-a56a-3d2b122bc66e/verify

![微信图片_20210521214409](https://user-images.githubusercontent.com/61954441/119146947-b711af80-ba7d-11eb-96fc-4e4a491c34fc.png)

照片也对应的上传了，然后请求的时候就出现Something went wrong,please try again这个错误信息,跪求大神这个是什么原因

**Comment by pospielov** [comment]
I checked and it looks like this is a critical bug. 
We already fixed it, so you can build CompreFace yourself from the 0.5.1 branch, we will release a hotfix version during the next week.
Sorry for the inconvenience.

**Reply by yang1158280920** [reply]
It’s okay

**Reply by yang1158280920** [reply]
![image](https://user-images.githubusercontent.com/61954441/119247275-ae72c380-bbba-11eb-95d6-4015fa8c1a13.png)
![image](https://user-images.githubusercontent.com/61954441/119247288-c8140b00-bbba-11eb-8143-3aa9bf14fe59.png)
这两个API上都没有注明image_id写在哪里或在哪个url后面

**Reply by pospielov** [reply]
Hi, it was definitely a typo, thanks, I fixed it!

**Comment by pospielov** [comment]
Hi, please check `Verify Faces from a Given Image` in 0.5.1 version, it should work now

**Reply by yang1158280920** [reply]
thank you
